packages/main/src/TimePickerInternals.ts

```diff
@@ -99,26 +124,9 @@ class TimePickerInternals extends UI5Element {
 	@property()
 	formatPattern!: string;
 
-	/**
```

Comment:
These APIs are in a private component and they will not affect any public APIs, right?

Reply:
yes, they receive their values from TimePicker's corresponding properties in order to create interface parts (clocks/inputs) according to format set in the TimePicker.